packages/forms-web-app/src/routes/eligibility/granted-or-refused-permission.js

```diff
@@ -9,6 +9,12 @@ const {
 
 const router = express.Router();
 
+/* Get eligibility granted-or-refused-permission out page */
```

Comment:
I don't think we need these comments in the code to explain the routes, the should be explanatory enough from the route url